tests/TestCase.php

```diff
@@ -41,7 +41,7 @@ public static function prepareSession()
      *
      * @throws \UnexpectedValueException if the global driver_config_factory returns an invalid object
      */
-    private static function getConfig()
+    private static function getConfig(): AbstractConfig
```

Comment:
I figured that changing the signature of a private method should be BC safe enough,